File: app/src/main/java/pakutoma/iijmiocouponwidget/MainSettingsActivity.java
package pakutoma.iijmiocouponwidget;

import android.preference.PreferenceFragment;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import pakutoma.iijmiocouponwidget.R;

public class MainSettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsPreferenceFragment())
                .commit();
    }

    public static class SettingsPreferenceFragment extends PreferenceFragment{
        @Override
        public void onCreate(Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);
            addPreferencesFromResource(R.xml.activity_main_settings);
        }
    }
}

File: app/src/main/java/pakutoma/iijmiocouponwidget/activity/MainSettingsActivity.kt
package pakutoma.iijmiocouponwidget.activity

import android.os.Bundle

import pakutoma.iijmiocouponwidget.R
import android.preference.PreferenceFragment
import android.support.v7.app.AppCompatActivity


class MainSettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        fragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsPreferenceFragment())
                .commit()
    }

    class SettingsPreferenceFragment : PreferenceFragment() {
        override fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)
            addPreferencesFromResource(R.xml.activity_main_settings)
        }
    }
}